layouts/partials/fragments/list.html

```diff
@@ -77,16 +77,16 @@ <h5 class="col-12 col-md px-0">
                             </a>
                           </h5>
                         {{- end -}}
-                        {{- if and (not $self.Params.tiled) (ne $self.Params.display_date false) }}
+                        {{- if and (not $self.Params.tiled) (eq $self.Params.display_date false) }}
                           {{- range $content_page -}}
                             {{- partial "helpers/publish-date.html" (dict "root" . "background" $bg "badge" false) -}}
                           {{- end -}}
                         {{- end }}
                       </div>
                     </div>
-                    {{- if or (and $self.Params.tiled (ne $self.Params.display_date false)) .Params.categories -}}
+                    {{- if or (and $self.Params.tiled (eq $self.Params.display_date false)) .Params.categories -}}
                       <div class="col-12 pb-1 px-0">
-                        {{- if and $self.Params.tiled (ne $self.Params.display_date false) -}}
+                        {{- if and $self.Params.tiled (eq $self.Params.display_date false) -}}
```

Comment:
Shouldn't that just be `{{- if and $self.Params.tiled $self.Params.display_date -}}`